### ISSUE TITLE
Model performance

### ISSUE THREAD

**Issue by fmahoudeau** [developer0hye/PyTorch-Darknet53#3]
Hi, my understanding from the paper and the code on the Darknet repository is that both training and evaluation are performed in 256x256 pixels for all models shown in the table. This includes Resnet101 and Resnet152 (see https://github.com/pjreddie/darknet/blob/master/cfg/resnet101.cfg). You are using 224x224 and should see a leap after retraining in 256x256. Also why are you resizing the images to 256x256 at test time ? Is this standard practice ? I would just center crop images at test time.

**Comment by developer0hye** [comment]
Hi.

It is a very good question.

Actually, the original paper's authors trained and tested darknet53 with 256x256 image size.

But, the conventional methods like vgg,resnet and densenet trained and tested with 224x224 image size, so I trained and tested with 224x224 image size.

And I followed [this repository](https://github.com/pytorch/examples/blob/master/imagenet/main.py) for test.

I hope it helps you.

**Comment by fmahoudeau** [comment]
Okay, so I think you are being unfair to yourself when you wrote "I could not get the good results like the above table". It would be interesting to train with images cropped to 256x256 to be on par with the methodology used by the Darknet authors. 

Also it would be very appealing if your repository included pre-trained weights in 448x448. Not just to compare with the paper's results, but because many transfer learning tasks are done in higher resolution such as 512x512 (Human Pose estimation, Image Segmentation, etc...). Using a backbone pre-trained in only 224x224 is such a waste. So a well trained Darknet53_448 would be a great replacement for ResNets.

Lastly do you know what the authors applied in terms of normalization and augmentation ? I cannot find the information.

Kind regards

**Comment by developer0hye** [comment]
@fmahoudeau 
Yeah, I agree with you.

But... I have no time recently, so I will train this model with higher input image size if I finish my works.

I do not **exactly** know the augmentation method applied in original implementation.

You should refer to [darknet53.cfg](https://github.com/pjreddie/darknet/blob/master/cfg/darknet53.cfg).
```
...
min_crop=128 
max_crop=448
...
angle=7
hue=.1
saturation=.75
exposure=.75
aspect=.75
```
I guess they applied rotation(-7 ,+7 degree), color jittering, cropping and affine transformation.
